test/e2e/libpod_suite_test.go

```diff
@@ -0,0 +1,258 @@
+	}
+}
+
+//MakeOptions assembles all the podman main options
+func (p *PodmanTest) MakeOptions() []string {
+	return strings.Split(fmt.Sprintf("--root %s --runroot %s --runtime %s --conmon %s --cni-config-dir %s",
+		p.CrioRoot, p.RunRoot, p.RunCBinary, p.ConmonBinary, p.CNIConfigDir), " ")
+}
+
+// Podman is the exec call to podman on the filesystem
+func (p *PodmanTest) Podman(args []string) *PodmanSession {
+	podmanOptions := p.MakeOptions()
+	podmanOptions = append(podmanOptions, strings.Split(p.StorageOptions, " ")...)
+	podmanOptions = append(podmanOptions, args...)
+	fmt.Println(strings.Join(podmanOptions, " "))
+	command := exec.Command(p.PodmanBinary, podmanOptions...)
+	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+	if err != nil {
+		Fail(fmt.Sprintf("unable to run podman command: %s", strings.Join(podmanOptions, " ")))
+	}
+	return &PodmanSession{session}
+}
+
+// Cleanup cleans up the temporary store
+func (p *PodmanTest) Cleanup() {
+	// does nothing yet
+}
+
+// GrepString takes session output and behaves like grep. it returns a bool
+// if successful and an array of strings on positive matches
+func (s *PodmanSession) GrepString(term string) (bool, []string) {
+	// not implement
+	return false, nil
+}
+
+// Pull Images pulls multiple images
+func (p *PodmanTest) PullImages(images []string) error {
+	for _, i := range images {
+		p.PullImage(i)
+	}
+	return nil
+}
+
+// Pull Image a single image
+// TODO should the timeout be configurable?
+func (p *PodmanTest) PullImage(image string) error {
+	session := p.Podman([]string{"pull", image})
+	session.Wait(60)
+	Expect(session.ExitCode()).To(Equal(0))
+	return nil
+}
+
+// OutputToString formats session output to string
+func (s *PodmanSession) OutputToString() string {
+	return fmt.Sprintf("%s", s.Out.Contents())
+}
+
+// SystemExec is used to exec a system command to check its exit code or output
+func (p *PodmanSession) SystemExec(command string, args []string) (*PodmanSession, error) {
+	// not implemented yet
+	return nil, nil
+}
+
+// CreateArtifact creates a cached image in the artifact dir
+func (p *PodmanTest) CreateArtifact(image string) error {
+	imageName := fmt.Sprintf("docker://%s", image)
+	systemContext := types.SystemContext{
+		SignaturePolicyPath: p.SignaturePolicyPath,
+	}
+	policy, err := signature.DefaultPolicy(&systemContext)
+	if err != nil {
+		return errors.Errorf("error loading signature policy: %v", err)
+	}
+	policyContext, err := signature.NewPolicyContext(policy)
+	if err != nil {
+		return errors.Errorf("error loading signature policy: %v", err)
+	}
+	defer func() {
+		_ = policyContext.Destroy()
+	}()
+	options := &copy.Options{}
+
+	importRef, err := alltransports.ParseImageName(fmt.Sprintf("docker://%s", imageName))
+	if err != nil {
+		return errors.Errorf("error parsing image name %v: %v", image, err)
+	}
+
+	exportTo := filepath.Join(p.ArtifactPath, "image")
+	exportRef, err := alltransports.ParseImageName(exportTo)
```

Comment:
You'll need to prepend "dir:" to `exportTo` here.

Reply:
fixed